Source/Wikitivty.CreateInstanceSetting/PostInstallEventHandler.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Net;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using kCura.EventHandler;
+using kCura.EventHandler.CustomAttributes;
+using kCura.Relativity.Client;
+using Relativity.API;
+using Relativity.Services.Interfaces.InstanceSetting;
+using Relativity.Services.Interfaces.InstanceSetting.Model;
+using Relativity.Services.Objects;
+
+namespace Wikitivty.CreateInstanceSetting
+{
+	[kCura.EventHandler.CustomAttributes.RunOnce(true)]
+	[kCura.EventHandler.CustomAttributes.Description("Wikitivity - Create WebAPI Instance Setting")]
+	[System.Runtime.InteropServices.Guid("45626737-dd36-437b-adc4-6502e8882a34")]
+	public class PostInstallEventHandler : kCura.EventHandler.PostInstallEventHandler
+	{
+		public override Response Execute()
+		{
+			// Update Security Protocol
+			ServicePointManager.SecurityProtocol = SecurityProtocolType.Ssl3 | SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;
+
+			//Construct a response object with default values.
+			kCura.EventHandler.Response retVal = new kCura.EventHandler.Response();
+			retVal.Success = true;
+			retVal.Message = string.Empty;
+			try
+			{
+				IInstanceSettingManager proxy = Helper.GetServicesManager().CreateProxy<IInstanceSettingManager>(ExecutionIdentity.System);
+				int operationResults = CreateWikitivityWebAPIInstanceSetting(proxy, retVal).Result;

```

Comment:
I would first check if the instance key exists. Only if it doesn't exist, I would proceed to create the instance setting.